src/Chassis/Env/EnvSwitcher.cs

```diff
@@ -27,8 +27,13 @@ internal void RegisterTypes(ContainerBuilder builder)
 
         internal TService Resolve(IComponentContext cxt)
         {
-            var t = _maps[AppEnv.ENV];
-            return cxt.Resolve(t) as TService;
+            if (_maps.ContainsKey(AppEnv.ENV))

```

Comment:
you won't ever get `null` 

https://github.com/mag-dev/chassis/blob/master/src/tests/Chassis.UnitTests/Env/AppEnvTests.cs#L46